Mlem/App/Views/Root/Tabs/Feeds/SubscriptionListItemView.swift

```diff
@@ -19,9 +19,14 @@ struct SubscriptionListItemView: View {
     let sectionIndicesShown: Bool
     
     var body: some View {
-        NavigationLink(.community(community)) {
-            HStack(spacing: 15, content: label)
-        }
+        MultiplatformView(phone: {
+            NavigationLink(.community(community), label: label)
+        }, pad: {
+            Button(action: {
+                navigation.root = .community(community)
+            }, label: label)
+                .buttonStyle(EmptyButtonStyle())
+        })
```

Comment:
Can this use the `multiplatformButton` defined in `SubscriptionListView`? (extracted to its own full view obv)

Reply:
Sure